web/src/utilities/analyticsVis.ts

```diff
@@ -0,0 +1,121 @@
+import { Dataset, DetectedStructure } from '@/generatedTypes/AtlascopeTypes';
+import { computed } from '@vue/composition-api';
+import geo from 'geojs';
+import { GeoJSLayer } from './composableTypes';
+import store from '../store';
+import { centroidStringToCoords, NucleusGlandDistance } from './utiltyFunctions';
+
+interface StructurePoint {
+  x: number;
+  y: number;
+  color: string;
+  struct: DetectedStructure;
```

Comment:
```suggestion
  structure: DetectedStructure;
```

I think its more clear to just use the full word here.

Reply:
Changed in [afaa324](https://github.com/atlascope/atlascope/pull/193/commits/afaa32436d67da62b3e0d4ccf9e2adb104bb86cc)